README.md

```diff
@@ -1,20 +1,25 @@
 # JQuery/Ajax Quiz
 
-Write your answer below each question. 
+Write your answer below each question.
 
 ### Question 1
-(a) What is a click handler? 
+(a) What is a click handler?
+It binds an event handler to a click event - e.g, it connects to something that does something when the user clicks
 
-(b) Where in your JS file should you put it, and WHY? 
+(b) Where in your JS file should you put it, and WHY?
+It should be in your (document).ready function
+So it can access all functions/variables elsewhere
 
 ### Question 2
-If you get the error `$ is undefined`, what does that mean and what could you do to fix it? 
+If you get the error `$ is undefined`, what does that mean and what could you do to fix it?
```

Comment:
Seems like you skipped Q2. If you're getting `$ is undefined`, it means you haven't loaded jQuery correctly.